server/routes/api/stats.py

```diff
@@ -59,22 +61,20 @@ def stats(stats_var):
     return get_stats_wrapper('^'.join(place_dcids), stats_var)
 
 
-@cache.memoize(timeout=3600 * 24)  # Cache for one day.
-def get_statsinfo_wrapper(statsvars_string):
-    """Wrapper function to get stats information give multiple stats var.
+def get_stats_info(dcids):
```

Comment:
What's the result if the dcid isn't present?

Hm, I'm struggling to come up with a better name for this because info is so generic. Maybe it should be get_stats_url_fragment?

Reply:
if dcid is empty then this return empty dict. Updated the function name.